lib/ipmi_fru.c

```diff
@@ -832,11 +845,12 @@ read_fru_area_section(struct ipmi_intf * intf, struct fru_info *fru, uint8_t id,
 			break;
 		}
 		if (rsp->ccode) {
-			/* if we get C7 or C8  or CA return code then we requested too
-			* many bytes at once so try again with smaller size */
-			if ((rsp->ccode == 0xc7 || rsp->ccode == 0xc8 || rsp->ccode == 0xca) &&
-				(--fru_data_rqst_size > 8)) {
-				lprintf(LOG_INFO, "Retrying FRU read with request size %d",
+			/* if we get C7 or C8  or CA return code then we
+			 * requested too many bytes at once so try again with
+			 * smaller size */
```

Comment:
Almost the same here. The next line is just as wide as was the original comment.